examples/test-staggered/README.md

```diff
@@ -0,0 +1,11 @@
+test-staggered
+==============
+
+Test of staggered grids for wave equation
```

Comment:
True, couldn't think of a better word. After v4 is out, we'll have a test push and make it a proper test!